docker-compose.yml

```diff
@@ -135,29 +129,14 @@ services:
       - prysm-beacon-node
       - jaeger-tracing
     build:
-      context: ./prysm
-      dockerfile: Dockerfile.prysm
+      context: ./
+      dockerfile: ./prysm/Dockerfile.prysm
     command: >
-      validator
-      --accept-terms-of-use
-      --beacon-rpc-provider prysm-beacon-node:4000
-      --rpc
-      --grpc-gateway-host 0.0.0.0
-      --grpc-gateway-port 7500
-      --interop-num-validators=4
-      --interop-start-index=0
-      --force-clear-db
-      --chain-config-file=/config/chain-config.yml
-      --suggested-fee-recipient 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b
-      --enable-tracing
-      --tracing-endpoint http://jaeger-tracing:14268/api/traces
-      --tracing-process-name validator-node
+      validator --accept-terms-of-use --beacon-rpc-provider prysm-beacon-node:4000 --rpc --grpc-gateway-host 0.0.0.0 --grpc-gateway-port 7500 --interop-num-validators=4 --interop-start-index=0 --force-clear-db --chain-config-file=/shared/chain-config.yml --suggested-fee-recipient 0xa94f5374fce5edbc8e2a8697c15331677e6ebf0b --enable-tracing --tracing-endpoint http://jaeger-tracing:14268/api/traces --tracing-process-name validator-node
```

Comment:
nit: can we have this on multiple lines like before. It's harder to read.

Reply:
np, this is auto fomratted by VS code, I'll change it back